node/src/components/block_validator/tests.rs

```diff
@@ -234,16 +320,16 @@ struct ValidationContext {
     past_blocks: HashMap<u64, Block>,
     // blocks that will be "stored" during validation
     delayed_blocks: HashMap<u64, Block>,
-    deploys: HashMap<TransactionId, Deploy>,
-    transfers: HashMap<TransactionId, Deploy>,
+    transactions: HashMap<TransactionId, Transaction>,
+    transfers: HashMap<TransactionId, Transaction>,
```

Comment:
We might not need to separate transfers here anymore, either.

Reply:
We don't but I'd like to merge this in a follow up PR